content/scout/release-notes/platform.md

```diff
@@ -2,6 +2,8 @@
 title: Docker Scout release notes
 description: Learn about the latest features of Docker Scout
 keywords: docker scout, release notes, changelog, features, changes, delta, new, releases
+aliases:
+- /scout/release-notes/
 ---
 
 This page contains information about the new features, improvements, known
```

Comment:
Should we now point the CLI release notes to `/scout/release-notes/cli.md`?

Reply:
ah right there's a cross-reference on this page. Let me fix it